tests/test_gunicorn.py

```diff
@@ -121,14 +122,29 @@ def test_gunicorn_logger_access(environ, log, statsd_metrics):
     cfg.set('accesslog', '-')
     logger = gunicorn.GunicornLogger(cfg)
 
+    log[:] = []
```

Comment:
Could also `del log[:]` but I'm not sure that's any clearer (pub also intended).